.github/workflows/pr-dependencies-check.yml

```diff
@@ -0,0 +1,211 @@
+name: PR Dependency Check
+
+on:
+  workflow_dispatch:
+    inputs:
+      pr_number:
+        description: 'PR number or URL'
```

Comment:
I would limit it just to PR numbers for now, as passing `https://github.com/pytorch/test-infra/pull/4747` as an argument does not yield expected results